my_dual_robot_cell/my_dual_robot_cell_description/urdf/my_dual_robot_cell_macro.xacro

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://ros.org/wiki/xacro">
+  <xacro:include filename="$(find ur_description)/urdf/ur_macro.xacro"/>
+
+  <xacro:macro name="my_dual_robot_cell" params="
+    parent
+    *origin
+
+    alice_ur_type
+    alice_joint_limits_parameters_file
+    alice_kinematics_parameters_file
+    alice_physical_parameters_file
+    alice_visual_parameters_file
+
+    bob_ur_type
+    bob_joint_limits_parameters_file
+    bob_kinematics_parameters_file
+    bob_physical_parameters_file
+    bob_visual_parameters_file
+    ">
+
+    <joint name="table_base_joint" type="fixed">
+      <xacro:insert_block name="origin" />
+      <parent link="${parent}" />
+      <child link="table" />
+    </joint>
+
+    <link name="table">
+      <visual>
+        <geometry>
+          <mesh filename="package://my_dual_robot_cell_description/meshes/fzi_table.dae"/>
+        </geometry>
+        <origin rpy="0 0 0" xyz="0.75 0.5 -1"/>
+      </visual>
+      <collision>
+        <geometry>
+          <mesh filename="package://my_dual_robot_cell_description/meshes/fzi_table.dae"/>
+        </geometry>
+        <origin rpy="0 0 0" xyz="0.75 0.5 -1"/>
+      </collision>
+    </link>
+
+    <link name="monitor">
+      <visual>
+        <geometry>
+          <mesh filename="package://my_dual_robot_cell_description/meshes/171005_Motek_Monitor_L.dae"/>
+        </geometry>
+      </visual>
+      <collision>
+        <geometry>
+          <mesh filename="package://my_dual_robot_cell_description/meshes/171005_Motek_Monitor_L.dae"/>
+        </geometry>
+      </collision>
+    </link>
+    <joint name="base_to_monitor" type="fixed">
+      <parent link="table"/>
+      <child link="monitor"/>
+      <origin xyz="0.2 0.6 0"/>
+    </joint>
+
+    <material name="white">
+      <color rgba="1 1 1 1"/>
+    </material>
+
+    <link name="wall">
+      <visual>
+        <geometry>
+          <mesh filename="package://my_dual_robot_cell_description/meshes/wall.dae"/>
+        </geometry>
+      </visual>
+      <collision>
+        <geometry>
+          <box size="4.44 0.1 2.5"/>
+        </geometry>
+      </collision>
+    </link>
+    <joint name="base_to_wall" type="fixed">
+      <parent link="table"/>
+      <child link="wall"/>
+      <origin xyz="1.25 1.5 0.5"/>
+    </joint>
+
+
+    <!--Creates the mounts to attach the robots to the table-->
+    <link name="alice_robot_mount"/>
+    <joint name="base_to_alice_robot_mount" type="fixed">
+      <parent link="table"/>
+      <child link="alice_robot_mount"/>
+      <origin xyz="0.3 0.85 0" rpy="0 0 ${pi}" />
```

Comment:
Maybe the placement isn't ideal since it is right behind the computer screen on the table.